src/holofuel/__integration_tests__/holo/snapshots/.gitignore

```diff
@@ -0,0 +1,3 @@
+*
+
+!.gitignore
```

Comment:
@JettTech - adding this file means that the snapshots directory will be created when you check out this branch, but nothing in the directory (except this file itself) will be committed